app/views/users/emails/show.html.erb

```diff
@@ -12,6 +12,7 @@
           form: f,
           name: :email,
           label: t('forms.registration.labels.email'),
+          input_html: { autocomplete: 'off' },
```

Comment:
Curious if there might be a way to get test coverage for this. I wondered if we could do something in Chromedriver to simulate the autofill, but it looks like [the devtools support for that](https://chromedevtools.github.io/devtools-protocol/tot/Autofill/) is pretty limited.

Maybe we could have a check in the view spec for the attribute being present on the input?